### PR TITLE
Change md convention for word

### PR DESCRIPTION
**Pull request by 7Pawns**
Just a tiny change in the docs, so the text will follow conventions of other instances of it.

### CODE REVIEW

**Comment by stmcginnis** [comment]
There is a difference in the concept of "role binding", and the k8s object of a `RoleBinding`. I believe the existing text is correct.

**Comment by dipesh-rawat** [comment]
@7Pawns Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by niranjandarshann** [comment]
I think there is no need of changes in the sentence . the `role binding` have the separate  meaning here whereas `RoleBinding `have different purpose in Kubernetes.
It is also required to get the easycla Signed toraise PR.

File: content/en/docs/reference/access-authn-authz/rbac.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ The name of a Role or a ClusterRole object must be a valid
 
 ### RoleBinding and ClusterRoleBinding
 
-A role binding grants the permissions defined in a role to a user or set of users.
+A RoleBinding grants the permissions defined in a role to a user or set of users.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/rbac.md:117]: I think the existing test is more correct. This intro paragraph address both `RoleBinding` and `ClusterRoleBinding` API objects, and explains the difference between them.

**Pull request closed without merging**